### ISSUE TITLE
Health 4.1.0 - Can't fetch other data, only steps

### ISSUE THREAD

**Issue by Nedimko123** [cph-cachet/flutter-plugins#574]
I copied the example code, and I can fetch the steps data, but when I fetch other data I get this:

```
I/flutter (21117): requested: true
I/flutter (21117): Exception in getHealthDataFromTypes: Null check operator used on a null value
```
Using the latest Flutter and Dart version, and it is connected to Google Fit and it has some data at Google Fit

**Comment by Nedimko123** [comment]
I tried the 4+ versions, those 2 didn't work, so I tried this version:   health: ^3.4.4 and it worked

**Comment by CemTitor** [comment]
Iam having same issue, is there any other solution?

**Comment by CemTitor** [comment]
Fixed #575
